ClientApp/src/modules/common/components/InsuranceClientData/InsuranceClientData.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {Text, Container, Grid, GridItem} from '@chakra-ui/react';
+import {PersonIcon} from './InsuranceClientData.icons';
+import 'react-datepicker/dist/react-datepicker.css';
+
+interface Props { 
+    name: string;
+    data: string[];
```

Comment:
Combine "name" and "data" props into single object, for example "personInfo: PersonInfo". Define this type in file InsuranceClientData.types.ts